lib/modules/showImages.js

```diff
@@ -908,6 +908,16 @@ function shouldLinkBeDeferred(element) {
 const inText = element => !!element.closest('.md, .search-result-footer');
 
 async function checkElementForMedia(element) {
+	// Prevents relatively addressed anchor link urls from refreshing the page,
```

Comment:
Could you move this to the settings module which handles these clicks? If `showImages` is disabled, then this won't run.  The overhead won't be that bad.